tests/tso_cluster.go

```diff
@@ -173,7 +173,7 @@ func (tc *TestTSOCluster) WaitForPrimaryServing(re *require.Assertions, keyspace
 			}
 		}
 		return false
-	}, testutil.WithWaitFor(5*time.Second), testutil.WithTickInterval(50*time.Millisecond))
+	}, testutil.WithWaitFor(30*time.Second), testutil.WithTickInterval(100*time.Millisecond))
```

Comment:
Is it only a case problem?

Reply:
No, according to the test log, it's just because we created a lot of keyspace groups at one time and the election can not finish that fast. The log shows that the failure point has a 10-second gap with the election success which the condition waiting requires. I believe this could fix it.